static/js/ducks/sources.js

```diff
@@ -14,8 +18,15 @@ export function fetchSources(filterParams = {}) {
   return API.GET(`/api/sources?${queryString}`, FETCH_SOURCES);
 }
 
+export function fetchGroupSources(filterParams = {}) {
+  const params = new URLSearchParams(filterParams);
+  const queryString = params.toString();
+  return API.GET(`/api/sources?${queryString}`, FETCH_GROUP_SOURCES);
+}
+
 const initialState = {
   latest: null,
+  groupSources: null,
```

Comment:
do you need any of the other params here? pageNumber, lastPage, etc.? are those ever used?

Reply:
I think they are used in the sources list page. We are using the same reducer with a different case for the group sources page.